spec/support/flores_extensions.rb

```diff
@@ -1,6 +1,46 @@
 # encoding: utf-8
 require "flores/pki"
+require "flores/random"
+
 module Flores
```

Comment:
Well, I do this for 3 reasons.. 
- input beats is kind of weird beast to test and I have to create a lot of helpers for that..
- It easier to test in an actual plugins than having to manager multiples gems revision ;)
- I have access to rubygems for flores, but not on the github project itself so I cannot merge on it. (not sure if jarvis has access to it though, I haven't tested it.)

For monkey patch, I think its OK since we are controlling the flores library and its only for testing purposes, I will move it ASAP :tm: 

Well if you like this helper you should look for `let_tmp_file(:lol) { "Hello world" }` in https://github.com/logstash-plugins/logstash-input-beats/blob/master/spec/support/file_helpers.rb